components/mpas-ocean/src/Registry.xml

```diff
@@ -1305,6 +1355,10 @@
 					description="If true, $h^{hf}$ is included in the desired ALE thickness, and the prognostic equations for $D^{lf}$ and $h^{hf}$ are integrated in the code."
 					possible_values=".true. or .false."
 		/>
+                <nml_option name="config_disable_ALE" type="logical" default_value=".false." units="unitless"
+                                        description="flag to disable the ALE coordinate and revert to eulerian mode NOTE: only works for RK4 at present"
+                                        possible_values=".true. or .false."
```

Comment:
This is another really untested option that I would suggest we remove.  I don't think we want this finding its way to e3sm master